### ISSUE TITLE
react/jsx-indent-props

### ISSUE THREAD

**Issue by jarretmoses** [yannickcr/eslint-plugin-react#2828]
I just upgraded to v `7.21.4` And it seems this rule is not properly detecting the spacing in a couple places that had worked before this version. (The `@typescript-eslint/indent` I also have is working fine). I attached the photo where its not correctly noticing the spacing I have as it is fine as is and I am doing very similar types of spacing in my codebase where this rul is not complaining. This kinda exposed the fact it seems like I have redundant rules and can just rely on the @typescript-eslint rule but I figured I'd put this here anyways.
<img width="1032" alt="Screen Shot 2020-10-14 at 4 04 32 PM" src="https://user-images.githubusercontent.com/4745679/96040083-b5bd3c00-0e37-11eb-955d-1e2df8059e5b.png">

```
export const DKListingDetailItem = ({
  label,
  text,
  classNames = '',
  testid = '',
  children,
}: Props) => {
  const detail = typeof text === 'string'
    ? <p>{text}</p>
    // eslint-disable-next-line react/no-array-index-key
    : text.map((textDetail, i) => <p key={`textDetail${i}`}>{textDetail}</p>);

  return (
    <div
      className={`dk-listing-detail-item__section-container ${classNames}`.trim()}
      data-testid={testid}
    >
      <h4 className='dk-listing-detail-item__section-label'>{label}</h4>
      {detail}
      {children}
    </div>
  );
};
```

**Comment by jarretmoses** [comment]
An interesting note. If I remove the ternary above the error goes away.

**Comment by ljharb** [comment]
I believe this is fixed by #2826.

**Comment by jarretmoses** [comment]
ah thanks @ljharb Ill close this and wait for the next release to confirm.
